docs/development/quickstart.md

```diff
@@ -65,9 +65,34 @@ you should be on your way:
 The Test Pilot add-on is a key component of this project - among other things,
 it communicates with the site and grants the ability to enable & disable
 experiments. Setting up your browser with the steps above should make it easier
-for you to get it working.
+for you to get it working. 
 
 ## Windows hosts
 
-**Help wanted**: Getting things working on Windows may be similar to OS X,
-but the team has little experience with that environment.
+After installing Node.js for Windows, run these commands to get started: 
+
+```cmd
+:: Set up add-on environment and build an unsigned package
+cd addon
+npm install
+npm run package-win
+
+:: Set up frontend web site environment
+cd ..
+npm install
+```
+
+Now, open command prompt with admin privileges and run this:
+
+```cmd
+:: Add hostname alias to /etc/hosts and start up dev webserver
+echo 127.0.0.1 testpilot.dev >> %WINDIR%\System32\Drivers\Etc\Hosts
+```
+
+Go back to the previous command prompt window and run
```

Comment:
I'm a bit confused at why there are two command prompts here. Does it require admin privileges to add to hosts? Is there an easier way to do that than a new window??